src/training_data.py

```diff
@@ -77,16 +77,16 @@ def load_luis_data(self,filename):
         warnings.warn(
         """LUIS data may not always be correctly imported because entity locations are specified by tokens.
         If you use a tokenizer which behaves differently from LUIS's your entities might not be correct""")
-        data = json.loads(open(filename,'rb').read())
+        data = json.loads(codecs.open(filename, encoding='utf-8').read())
         for s in data["utterances"]:
-            text = unicode(s.get("text"))
-            tokens = [t.decode('utf-8') for t in self.tokenizer.tokenize(text)]
+            text = s.get("text")
+            tokens = [t for t in self.tokenizer.tokenize(text)]
             intent = s.get("intent")
             entities = []
             for e in s.get("entities") or []:
                 i, ii = e["startPos"], e["endPos"]+1
                 #print(u"full text:  {0}".format(text))
-                val = u"\s*".join([unicode(s) for s in tokens[i:ii+1]])
+                val = u"\s*".join([s for s in tokens[i:ii+1]])
```

Comment:
please fix this - see comment further down. should rename val to something else (like `_regex` maybe). `value` should be the actual value